mod/wikirate/lib/javascript/script_collapse.js.coffee

```diff
@@ -88,16 +88,16 @@ findCollapseTarget = (toggle) ->
     else
       $toggle.parent()
 
-  target = $toggle.data('target') || '.collapse'
+  if $toggle.attr("href") != null
+    return $($toggle.attr("href"))
+  target = $toggle.attr("href") || $toggle.data('target') || '.collapse'
   if $toggle.data("collapse")?
     target += $toggle.data("collapse")
-
-  if $toggle.find(target).length
+  else if $toggle.find(target).length
```

Comment:
you are right, line 93 and line 96 are wrong. I think I forgot to delete/redo it. My intention is only to add these two lines only. 
`  if $toggle.attr("href") != null
    return $($toggle.attr("href"))
`
Above snippet enables `data-collapse-text-in/out` to work for bootstrap collapse button where the target value is declared in `href` instead of `data-target`. In my opinion, declaring _target_ value on `href` right way instead of in `data-target` attribute.